### ISSUE TITLE
Make a new Nengo GUI release

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo-gui#988]
I just pip installed Nengo GUI and noticed that the model that pops up is

```python
import nengo
import nengo_spa as spa



model = spa.Network()
with model:

    stim = spa.Transcode('X', output_vocab=32)
```

Looking in this repo, I don't think that this change every got committed to master, so what likely happened is that `default.py` was changed locally (i.e., `git status` would have shown changes) when the PyPI release was made. Since this is what pops up when a new user start the GUI, and the chances that they have `nengo_spa` installed are super low, I think it's pretty important to fix quickly.

**Comment by tbekolay** [comment]
Can confirm this is fixed in 0.4.3, thanks!

**Comment by jgosmann** [comment]
Another reason to fix #878?
